k8s/plugin/README.md

```diff
@@ -205,57 +205,44 @@ Volume 0c08667c-8b59-4d11-9192-b54e27e0ce0f Scaled Successfully 🚀
 kubectl-mayastor-dump
 Supportability tool collects state & log information of services and dumps it to a tar file
 
-USAGE:
-    kubectl-mayastor dump [OPTIONS] <SUBCOMMAND>
-
-OPTIONS:
-    -d, --output-directory-path <OUTPUT_DIRECTORY_PATH>
-            Output directory path to store archive file [default: ./]
-
-    -e, --etcd-endpoint <ETCD_ENDPOINT>
-            Endpoint of ETCD service, if left empty then will be parsed from the internal service
-            name
-
-    -h, --help
-            Print help information
-
-    -j, --jaeger <JAEGER>
-            Trace rest requests to the Jaeger endpoint agent
-
-    -k, --kube-config-path <KUBE_CONFIG_PATH>
-            Path to kubeconfig file
-
-    -l, --loki-endpoint <LOKI_ENDPOINT>
-            Endpoint of LOKI service, if left empty then it will try to parse endpoint from Loki
-            service(K8s service resource), if the tool is unable to parse from service then logs
-            will be collected using Kube-apiserver
-
-    -n, --namespace <NAMESPACE>
-            Kubernetes namespace of mayastor service [default: mayastor]
-
-    -o, --output <OUTPUT>
-            The Output, viz yaml, json [default: none]
-
-    -r, --rest <REST>
-            The rest endpoint to connect to
-
-    -s, --since <SINCE>
-            Period states to collect all logs from last specified duration [default: 24h]
-
-    -t, --timeout <TIMEOUT>
-            Specifies the timeout value to interact with other modules of system [default: 10s]
-
-SUBCOMMANDS:
-    help       Print this message or the help of the given subcommand(s)
-    node       Collects information about particular node matching to given node ID
-    nodes      Collects information about all nodes
-    pool       Collects information about particular pool and its descendants matching to given
-                   pool ID
-    pools      Collects information about all pools and its descendants (nodes)
-    system     Collects entire system information
-    volume     Collects information about particular volume and its descendants matching to
-                   given volume ID
-    volumes    Collects information about all volumes and its descendants (replicas/pools/nodes)
+Usage: kubectl-mayastor dump [OPTIONS] <COMMAND>
```

Comment:
Isn't "dump" qualify as the actual COMMAND here? I mean, should we leave it as SUBCOMMAND?

Reply:
This is the actual output that we get on `kubectl mayastor dump --help`. Copied from there. Will the latest version of `clap` the document generated is like this.

Reply:
Resolving. As the option has been removed.